content/docs/developing-operators/tasks.md

```diff
@@ -177,13 +177,97 @@ Once our Nginx server is up and running we should be able to read a pearl of cow
 
 **Note:**
 
-- File generating Pod has to be side-effect free (meaning side-effects that are observable outside of the container like a 3rd party API call) as the container might be executed multiple times on failure. A `restartPolicy: OnFailure` is used for the pipe-pod
+- File generating Pod has to be side effect free (meaning side effects that are observable outside of the container like a 3rd party API call) as the container might be executed multiple times on failure. A `restartPolicy: OnFailure` is used for the pipe-pod
 - Only files <1Mb are applicable to be stored as ConfigMap or Secret. A pipe-task will fail should it try to copy files >1Mb
 - As of KUDO 0.9.0 pipe-artifacts can only be used within the same plan they were generated
 - You can pipeline artifacts by creating them in one pipe-task and mounting in a subsequent one
 
 Full cowsay-operator can be found in the [KUDO operators repo](https://github.com/kudobuilder/operators/tree/master/repository/cowsay).
 
+## KudoOperator-Task
+
+A `KudoOperator` task allows you to specify a dependency on other operators. While dependencies in general is a complicated topic, `KudoOperator` itself is about installation dependencies e.g. your operator and all the operators it depends on (including transitive dependencies) will be installed and/or removed as one unit.
```

Comment:
> Removal isn't implemented yet.

what do you mean? If you remove the parent instance, all the children (and their resources) are also automatically removed thx to the magic of the owner references!

Reply:
You are right, I forgot about this behavior. On the other hand this is true only for instances. So this is more of a nitpick, because it isn't clear what "operator" means in this context. Let's use "operator instance" instead. Operator and OperatorVersion resources of dependencies won't be removed if one removes a parent Operator or OperatorVersion.